lib/actions/BackendAction.js

```diff
@@ -124,11 +121,24 @@ class BackendAction {
     })
   }
 
-  _writePipelines (pipelines, cb) {
-    fsEx.emptyDir(this.pipelinesFolder, (err) => {
+  _writeLocalPipelines (cb) {
+    async.parallel([
+      cb => this.dcClient.downloadPipelines(AppSettings.getInstance().getId(), false, cb),
+      cb => this.dcClient.downloadPipelines(AppSettings.getInstance().getId(), true, cb)
+    ], (err, pipelines) => {
```

Comment:
this will result in the writing of pipelines waiting for *both* trusted and untrusted pipelines to download!
I suggest adding the writing to the cb of the download so that writing to disk can beginn immediatly

Reply:
Yepp, also for better readability: can you rename the callbacks from async.parallel from cb to sth. like pcb?